controllers/comments.controllers.js

```diff
@@ -14,4 +14,21 @@ function getCommentsByArticleId(req,res,next){
 }
 
 
-module.exports = {getCommentsByArticleId}
\ No newline at end of file
+
+function postComments(req,res,next){
+    const {username,body} = req.body
+    const {article_id} = req.params
+    
+    Promise.all(
+        [insertComments(username,body,article_id),checkArticleExists(article_id)]
```

Comment:
article_d is a foreign key, so a separate check that the article exists should not be necessary as a psql error will be thrown.